pom.xml

```diff
@@ -37,6 +37,7 @@
     <!-- Dependency versions overriding -->
     <junit-jupiter.version>5.5.2</junit-jupiter.version>
     <mockito.version>3.1.0</mockito.version>
+    <bnd.version>4.3.0</bnd.version>
```

Comment:
Just a minor comment here: this should go three lines up right after `byte-buddy` as it's not overriding a dependency from `assertj-parent-pom`

Reply:
will do!